src/main/java/com/sap/sgs/phosphor/fosstars/model/value/Language.java

```diff
@@ -40,6 +40,9 @@ public static Language parse(String string) {
         return C_SHARP;
       case "F#":
         return F_SHARP;
+      case "go":
```

Comment:
As `string.toUpperCase()` is used the cases should be `GO` and `GOLANG`.

Reply:
Good catch! I've fixed that.